apps/bfd-pipeline/bfd-pipeline-app/src/test/java/gov/cms/bfd/pipeline/app/PipelineManagerIT.java

```diff
@@ -22,296 +22,352 @@
 import org.junit.Assert;
 import org.junit.BeforeClass;
 import org.junit.Ignore;
+import org.junit.Rule;
 import org.junit.Test;
+import org.junit.rules.TestWatcher;
+import org.junit.runner.Description;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 /** Integration tests for {@link PipelineManager}, {@link PipelineJobRecordStore}, and friends. */
 public final class PipelineManagerIT {
   private static final Logger LOGGER = LoggerFactory.getLogger(PipelineManagerIT.class);
 
+  @Rule
+  public TestWatcher testCaseEntryExitLogger =
+      new TestWatcher() {
+        /** @see org.junit.rules.TestWatcher#starting(org.junit.runner.Description) */
+        @Override
+        protected void starting(Description description) {
+          LOGGER.info("{}: starting.", description.getDisplayName());
```

Comment:
Always love logging that stuff; altho I try to put about 100 * and BIGGEST ALL-CAPS I can find!